.github/workflows/wheel.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     env:
       CIBW_TEST_COMMAND: pytest --pyargs numcodecs
       CIBW_TEST_REQUIRES: pytest
-      CIBW_SKIP: "*27* pp* *35*"
+      CIBW_SKIP: "pp*"
```

Comment:
AFAICT cibuildwheel still supports Python 3.6, correct? Is there anyway for us to state here that we still support it? Just want to make sure if an update to cibuildwheel drops Python 3.6 from cibuildwheel that we catch it (admittedly maybe we should be looking to drop Python 3.6 anyways just want to make sure we do that explicitly as opposed to implicitly due to tooling changes).

Reply:
Ok sounds like there isn't a good way to do this, but that's ok since Numcodecs will most likely drop Python 3.6 before cibuildwheel does.